scripts/bootstrap_golden.py

```
@@ -35,86 +45,256 @@
+    neighbors = []
+    for p in result.points:
+        payload = p.payload or {}
+        if p.score < min_score or payload.get("page_id") == target.get("page_id"):
+            continue
+        neighbors.append(payload)
+        if len(neighbors) >= limit:
+            break
+    return neighbors
+
+
+def _record(qid: str, query: str, domain: str | None, answerable: bool, chunk_ids: list[str]) -> tuple[dict, dict]:
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)